admin/partials/admin-monk-post-meta-box-field-render.php

```diff
@@ -80,13 +78,43 @@
 						</option>
 				<?php endif; endforeach; ?>
 			</select>
-			<button class="monk-submit-translation button"><?php _e( 'Ok', 'monk' ); ?></button>
-			<a class="monk-cancel-submit-translation hide-if-no-js button-cancel"><?php _e( 'Cancel', 'monk' ); ?></a>
+			<button class="monk-submit-translation button"><?php esc_html_e( 'Ok', 'monk' ); ?></button>
+			<a class="monk-cancel-submit-translation hide-if-no-js button-cancel"><?php esc_html_e( 'Cancel', 'monk' ); ?></a>
 		<?php endif; ?>
 	</div>
 	<ul class="monk-translated-to">
 		<li>
 			<?php echo esc_html( $monk_languages[$post_default_language]['name'] ); ?>
+
+			<!-- 
+				Gives the option to alter the current post language
+				When the user select this feature,
+				the option containig translations is updated
+			-->
+			<a class="edit-post-status hide-if-no-js">
+				<span aria-hidden="true" class="monk-change-language">
+					<?php esc_html_e( 'Change', 'monk' ); ?>
+				</span>
+				<span class="screen-reader-text"><?php esc_html_e( 'Change current language', 'monk' ); ?></span>
+			</a>
+			<div class="monk-change-current-language">
+				<?php if ( $translation_counter !== count( $active_languages ) ) : ?>
+					<select name="monk_new_language">
+						<option value=""><?php esc_html_e( 'Choose language', 'monk' ); ?></option>
+						<?php
+							foreach ( $active_languages as $lang_code ) :
+								if ( array_key_exists( $lang_code, $monk_languages ) && ! array_key_exists( $lang_code, $post_translations ) ) :
+									$lang_name = $monk_languages[$lang_code]['name'];
+						?>
+								<option  value="<?php echo esc_attr( $lang_code ); ?>"/>
+									<?php echo esc_html( $lang_name ); ?>
+								</option>
+						<?php endif; endforeach; ?>
+					</select>
+					<button class="button"><?php esc_html_e( 'Ok', 'monk' ); ?></button>
```

Comment:
When clicking on `Ok` button, we'll need change current post language label to the new one.